src/ngrok/index.ts

```diff
@@ -170,28 +176,36 @@ export const start = async () => {
 
 export const stop = async () => {
   const api = getApi();
-  const response = ((await api.get("api/tunnels")) as unknown) as string;
-  const tunnels = (JSON.parse(response) as TunnelsResponse).tunnels;
+  const tunnels = await getActiveTunnels(api);
   if (tunnels.length > 0) {
     const tunnel = await window.showQuickPick([
       "All",
       ...tunnels.map((t) => t.public_url),
     ]);
     if (tunnel === "All") {
       await disconnect();
-      window.showInformationMessage("All ngrok tunnels disconnected.");
+      await kill();
+      window.showInformationMessage(
+        "All ngrok tunnels disconnected. ngrok has been shutdown."
+      );
     } else if (typeof tunnel !== "undefined") {
       await disconnect(tunnel);
-      window.showInformationMessage(`ngrok tunnel ${tunnel} disconnected.`);
+      let message = `ngrok tunnel ${tunnel} disconnected.`;
+      if ((await getActiveTunnels(api)).length === 0) {
+        await kill();
+        message = `${message}. ngrok has been shutdown.`;
```

Comment:
There is already period in the `message` declaration :)

Reply:
Good spot!